include/libmesh/rebuild_makefile.sh

```diff
@@ -30,16 +30,22 @@ cat <<EOF > Makefile.am
 # include the magic script!
 EXTRA_DIST = rebuild_makefile.sh
 
-BUILT_SOURCES = $built_sources
-
-DISTCLEANFILES = \$(BUILT_SOURCES)
-
 EOF
 
+echo -n "BUILT_SOURCES =" >> Makefile.am
+for built_src in $built_sources ; do
+    echo " \\" >> Makefile.am
+    echo -n "        "$built_src >> Makefile.am
+done
+
+echo >> Makefile.am
+echo >> Makefile.am
```

Comment:
Yup.  Seems to be the shortest completely-portable way to do it.  There's a bash/zsh/ksh idiom for simply embedding \n in a string, which even seems to work in dash (/bin/sh on Debian) but it's not POSIX compliant and I'd hate to have our stuff break as soon as someone decides to make their sh even stricter.